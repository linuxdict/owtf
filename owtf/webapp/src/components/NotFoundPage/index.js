/*
 * Component to show if page not found.
 */
import React from 'react';

export default class NotFoundPage extends React.Component {
  render() {
    return (
      <p>Page Not Found</p>
    );
  }
}
