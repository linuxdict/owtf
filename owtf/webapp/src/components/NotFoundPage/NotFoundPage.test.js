import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from './index';
import '../../setupTests';
import toJson from 'enzyme-to-json';

const setUp = (props={}) => {
    const component = shallow(<NotFoundPage {...props} />);
    return component;
};

describe('NotFoundPage Component', () => {

    let component;
    beforeEach(() => {
      component = setUp();
    });

    it('Should render without errors', () => {
      const tree = toJson(component);
      expect(component.find('p').length).toBe(1);
      expect(component.find('p').text()).toBe('Page Not Found');
      expect(tree).toMatchSnapshot();
    });


});
