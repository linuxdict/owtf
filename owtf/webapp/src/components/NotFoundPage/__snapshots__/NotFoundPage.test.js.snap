// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component Should render without errors 1`] = `
<p>
  Page Not Found
</p>
`;
